pkg/cmd/cli/backuplocation/create.go

```diff
@@ -78,6 +82,7 @@ func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&o.Provider, "provider", o.Provider, "name of the backup storage provider (e.g. aws, azure, gcp)")
 	flags.StringVar(&o.Bucket, "bucket", o.Bucket, "name of the object storage bucket where backups should be stored")
 	flags.StringVar(&o.Prefix, "prefix", o.Prefix, "prefix under which all Velero data should be stored within the bucket. Optional.")
+	flags.DurationVar(&o.BackupSyncPeriod, "backup-sync-period", defaultBackupSyncPeriod, "how often to ensure all Velero backups in object storage exist as Backup API objects in the cluster. Optional. Set this to `0s` to disable sync")
```

Comment:
Just wondering if we want to not set an explicit value for the BSL if the user doesn't specify the flag, rather than defaulting it to a minute here? If we did that, then the server-level setting would apply. The benefit is that if the user then changes the server-level setting, it would affect all BSLs that were not set up with an explicit sync period, whereas now, they're all getting an individual setting of 1m by default.

Reply:
Yes, this would actually be ideal. I'd like to make this change and the only issue i'm running into is how can I detect that the user didn't specify the flag? Atm when I set the default value to o.BackupSyncPeriod, like so:

` flags.DurationVar(&o.BackupSyncPeriod, "backup-sync-period", o.BackupSyncPeriod, "how often..."`
it gets parsed as 0s which is an issue since this means disable sync. Any suggestion on how to detect this flag wasn't provided?

Reply:
I *think* you can use this method: https://godoc.org/github.com/spf13/pflag#FlagSet.Changed

but I'm not 100% sure off the top of my head, would need to experiment

Reply:
Thanks @skriss, I modified to check if the user explicitly specified a backup sync period and only then set the backup sync period for the BSL. Otherwise if user does not specify the flag the BSL backup sync period is nil which will have the BSL use the server side backup sync.